package.json

```diff
@@ -31,7 +31,8 @@
   },
   "homepage": "https://github.com/DanielHreben/atlassian-connect-auth#readme",
   "dependencies": {
-    "atlassian-jwt": "^2.0.2"
+    "atlassian-jwt": "^2.0.2",
+    "axios": "^0.21.4"
```

Comment:
I had a look at a comparison and went with the most popular. 🤷‍♂️ 
But this is a valid concern and I have made some changes to have implementations of all these three client using peer dependencies so it will not bring them unnecessarily but they can be used optionally.

Reply:
After some more consideration I ended up defaulting to **got** and letting the axios and the fetch modules be optionally loaded if the user wants. In local tests the current structure worked just fine.